src/components/cards/YellowCard.tsx

```diff
@@ -0,0 +1,24 @@
+interface YellowCardProps {
+  title: string;
+  description: string;
+  className?: string;
+}
+
+const YellowCard = ({
+  title,
+  description,
+  className = "",
+}: YellowCardProps) => {
+  return (
+    <>
+      <div
+        className={`bg-yellow-200 flex justify-between px-5 py-5 ${className}`}
+      >
+        <h3 className="text-2xl font-semibold">{title}</h3>
+        <p className="text-sm w-3/4 text-center">{description}</p>
+      </div>
+    </>
+  );
```

Comment:
Great component since it is dynamic
![chip-n-dale-wink-thumbs-up-great-work-mg4x0nb7g1c0hbnh](https://github.com/EnAccess/co-design-FE/assets/77101006/5191636a-476b-410a-8b57-7596ec579511)